apps/aehttp/src/aehttp_logic.erl

```diff
@@ -160,3 +161,58 @@ get_top_blocks_time_summary(Count) ->
 connected_peers(Tag) -> aec_peers:connected_peers(Tag).
 
 blocked_peers() -> aec_peers:blocked_peers().
+
+get_top_blocks_gas_price_summary() ->
+    TopKeyHash = aec_chain:top_key_block_hash(),
+    case get_gens_min_gas_price(TopKeyHash, 480) of
+        Err = {error, _} ->
+            Err;
+        MinGasPrices ->
+            MinPriceN = fun(N) ->
+                            case lists:min(lists:sublist(MinGasPrices, N)) of
+                                undefined -> 0;
```

Comment:
I OTP25 lists:min([]) throws an exception, but never undefined?

Reply:
Moreover, minimum should then possibly be the consensus defined minimum?

Reply:
Returning 0 works for me as a way to distinguish
- is there empty space in blocks (then SDK should use minimum gas price)
- all blocks full, but transactions using minimum gas price (then SDK should increase the gas price a bit to ensure tx would be mined)
Otherwise I would ask for a flag indicating that.